{"metadata":{"role":"collectionGroup","roleHeading":"API Collection","modules":[{"name":"OpenAI"}],"title":"LocalizedError Implementations"},"schemaVersion":{"minor":3,"patch":0,"major":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/LocalizedError-Implementations"},"kind":"article","sections":[],"variants":[{"paths":["\/documentation\/openai\/apierrorresponse\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/errorDescription","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/failureReason","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/helpAnchor","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/recoverySuggestion"]}],"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"references":{"doc://OpenAI/documentation/OpenAI/APIErrorResponse/errorDescription":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/errorDescription","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"role":"symbol","url":"\/documentation\/openai\/apierrorresponse\/errordescription","abstract":[],"type":"topic","title":"errorDescription","kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/recoverySuggestion":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/recoverySuggestion","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"recoverySuggestion"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"role":"symbol","url":"\/documentation\/openai\/apierrorresponse\/recoverysuggestion","abstract":[],"type":"topic","title":"recoverySuggestion","kind":"symbol"},"doc://OpenAI/documentation/OpenAI":{"type":"topic","kind":"symbol","url":"\/documentation\/openai","role":"collection","title":"OpenAI","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","abstract":[]},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/helpAnchor":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/helpAnchor","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"helpAnchor"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}],"role":"symbol","url":"\/documentation\/openai\/apierrorresponse\/helpanchor","abstract":[],"type":"topic","title":"helpAnchor","kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/failureReason":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/failureReason","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"role":"symbol","url":"\/documentation\/openai\/apierrorresponse\/failurereason","abstract":[],"type":"topic","title":"failureReason","kind":"symbol"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"type":"topic","title":"APIErrorResponse","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"text":"APIErrorResponse","kind":"identifier"}],"url":"\/documentation\/openai\/apierrorresponse","navigatorTitle":[{"text":"APIErrorResponse","kind":"identifier"}],"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","role":"symbol","kind":"symbol"}}}